### ISSUE TITLE
Why Django 1.8 puts Jinja templates in a different directory

### ISSUE THREAD

**Issue by aaugustin** [niwibe/django-jinja#90]
Currently the documentation says:

> jinja2 are also templates, I don’t understand why put them in different directory

The answer is [here](https://github.com/django/deps/blob/master/accepted/0182-multiple-template-engines.rst#engine-selection). django-jinja implements option 3, while Django chose option 4.

You're welcome to disagree, but at least you should understand before disagreeing :-)

**Comment by niwibe** [comment]
Thanks for pointing me. I'll personally don't like the choiced approach by django.

Then, I think having a jinja2 backend in django and other third party backend (this one) can create confusion, and I don't know it was good idea including the jinja2 backend intro django. 

Firstly I have thinking maintains support for django <= 1.7 and recommend use native jinja2 backend if django 1.8 is used. But the native backend does not add support for django template filters, templatefilters autoloading, url/static resolve functions, etc... And I have decided add support to django 1.8 for django-jinja (the jinja2 directory is a "lesser evil").

What do you thing about it? What is the way what I should take for django-jinja? Any recommendations?

**Comment by aaugustin** [comment]
The built-in jinja2 backend will gain some features that django-jinja currently has, for instance better support for i18n. (I lacked time to do it properly in Django 1.8.) Other features like making all the DTL tags and filters available in Jinja2 template are less likely to be integrated.

I added support for multiple template engines (rather than just Jinja2) precisely to make it easy to write your own backend if Django's built-in backends don't do what you want. So I think there's a clear future for django-jinja: providing a slightly opinionated backend that makes the transition from the DTL easier.

Hopefully django-jinja will be simpler and easier to maintain once you drop support for Django < 1.8.

In order to help users of Django figure out whether django-jinja is a good choice for them, I would encourage you to document _why_ you diverge from the built-in Jinja2 backend.

Currently the documentation implies that Django simply lags behind django-jinja ("make it working like django_jinja is already working for years") but that isn't the whole story. There are also different design choices, as we're seeing here.
